package/MDAnalysis/analysis/msd.py

```diff
@@ -421,3 +456,46 @@ def _conclude_fft(self):  # with FFT, np.float64 bit prescision required.
                 positions[:, n, :]
             )
         self.results.timeseries = self.results.msds_by_particle.mean(axis=1)
+    
+
+    def _conclude_modified(self):
+        from collections import defaultdict
+        print("The Dump file has non-linear time spacing")
```

Comment:
Eventually, remove all `print` – if you want to show status messages, use the logger.